### ISSUE TITLE
Codekit reports Sass out-of-date when Sass is newer than Codekit's

### ISSUE THREAD

**Issue by jensimmons** [bdkjones/CodeKit2#382]
I just replaced the SSD drive in my computer and reinstalled everything from scratch (fun!). So I have Brand New Everything. I ran CodeKit for the first time, and it gave me this error message:

> Compass failed to run because your Mac has an older version of Sass and/or Compass installed that 
> conflicts with the newer versions in CodeKit. You must remove all versions of Sass below 3.3.rc6 and 
> all versions of Compass below 1.0.alpha18. Do this at the command line by running 'sudo gem 
> uninstall sass' and 'sudo gem uninstall compass'.

And I thought "hm, I doubt that." Sure enough, I'm running Sass 3.3.8, while Codekit is running Sass 3.3.7. 

It's too bad that I'll have to uninstall Sass. It'd be better if Codekit wasn't so picky. Maybe it's not supposed to be, and 3.3.8 (which is great than 3.3.rc6) should work. But I thought I'd file this ticket to say, if nothing else, at least the error message should take both realities into account (CLI-Sass might be older or newer), and not assume the CLI-copy is older.

![codekitisbeingdumb](https://cloud.githubusercontent.com/assets/108474/3414591/b6f71816-fe17-11e3-9fff-2544b7249668.jpg)

*Also, CodeKit is awesome.

**Comment by jensimmons** [comment]
Ah, wait!

I went to uninstall Sass and got this:

> Select gem to uninstall:
> 1. sass-3.2.19
> 2. sass-3.3.8
> 3. All versions

Apparently I had two versions of Sass installed, and one was older than 3.3. So I uninstalled just that one (3.2.19), and all is well. Everything works happy. 

![jensimmons com bash 8734 2014-06-27 12-34-07 2014-06-27 12-34-09](https://cloud.githubusercontent.com/assets/108474/3414731/3ba1eb80-fe19-11e3-948f-023eb67c78e7.jpg)

Sorry for bugging you with a ticket. I don't think there's anything to be done here...

**Comment by bdkjones** [comment]
Hey Jen,

Yea; what likely happened here is that you installed another gem (compass 0.12 perhaps) that listed the earlier version of Sass as a dependency, so it got installed without you explicitly specifying the older version.

Glad you got it solved though! Did your SSD fail or did you just upgrade? An SSD failure is the stuff of nightmares for me.

**Comment by guymeyer** [comment]
"An SSD failure is the stuff of nightmares for me."

This coming from the guy with four back ups last I checked

**Comment by jensimmons** [comment]
Yes, I do think installing Compass retro-installed old Sass. Dumb Compass / Ruby.

No, my SSD didn't fail (THANKFULLY!) — I just totally ran out of space. I'm trying to stretch my 2011 MacBook Air until Retina Thunderbolt Displays are announced, when I'll immediately buy said Retina Thunderbolt along with a new Retina MacBook Pro. I figured $300 was a great investment to bump from 256g to 480g in the meanwhile— freeing up scratch disk space to speed processing up. And while I was at it, I figured it was beyond time for a clean install. Buybye 3 years of apps and configs and who-knows-what weird leftovers piled on top of more weird leftovers. 

I just listened to you two on Shop Talk show. It was great to get to know you a bit! I hope all is going well with CodeKit.
